project_model_to_task/models/project.py

```diff
@@ -75,33 +75,26 @@ def goto_document(self):
 class IrActionActWindows(models.Model):
     _inherit = 'ir.actions.act_window'
 
-    def read(self, cr, uid, ids, fields=None, context=None,
-             load='_classic_read'):
-        if context is None:
-            context = {}
+    def read(self, fields, load='_classic_read'):
 
         def update_context(action):
             action['context'] = safe_eval(action.get('context', '{}'))
             action['context'].update({
-                'from_model': context.get('active_model'),
-                'from_id': context.get('active_id'),
+                'from_model': self._context.get('active_model'),
+                'from_id': self._context.get('active_id'),
```

Comment:
OK. "_context" is a private attribute, not partt of the official API, and should should not be used directly.
Please prefer using the public API `self.env.context`.